mandatory/2-function-creation.js

```diff
@@ -45,7 +95,11 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  for (let i = number; i > 0; i--) {
```

Comment:
nice use of a for loop!

Reply:
can you now re-write it with a while loop? (hint: for loops and while loops are exactly equivalent to each other: all for loops can be written as while loops and all while loops can be written as for loops)

Reply:
maybe like this 
function countReverse(number) {
  let i = number;
  while (i > 0) {
    console.log(i);
    i--;
  }
}